### ISSUE TITLE
Sql Package doesn't override all connection string properties when username and password are set

### ISSUE THREAD

**Issue by IvanAlekseev** [MicrosoftDocs/sql-docs#2430]
Regarding that page - https://docs.microsoft.com/en-us/sql/tools/sqlpackage

Looks like a bug was introduced in one of the recent versions of sqlpackage.exe
I am trying to publish a dacpac to azure database and previously when I set `TargetUser` and `TargetPassword` the whole connection string properties were overridden. But with the more recent version the publish command it still gets some bits from connection string specified in publish profile.

Here is my command line:
```
C:\Program Files\Microsoft SQL Server\150\DAC\bin\SqlPackage.exe" /Action:Publish /SourceFile:"d:\Database.dacpac" /TargetServerName:"*****" /TargetDatabaseName:"*****" /TargetUser:"***" /TargetPassword:"********" /Profile:"d:\Database.publish.xml" /p:AllowIncompatiblePlatform=true 
```

It was working when the sqlpackage.exe was coming from the folder: `C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\Common7\IDE\Extensions\Microsoft\SQLDB\DAC\140\SqlPackage.exe`
but it fails on newer build agent where we have version 150 installed.

Error I am getting:
```
Could not deploy package.
Unable to connect to target server '*****'. Please verify the connection information such as the server name, login credentials, and firewall rules for the target server.
Windows logins are not supported in this version of SQL Server.
```

If I remove TargetConnectionString from the publish profile, or change Integrated Security to False, then publish command works perfectly.

We did a comparison of decompiled versions of sqlpackage 140 and 150 and looks like for some reason builder.IntegratedSecurity is not set to false anymore when username and password are specified
![decompiled-comparison](https://user-images.githubusercontent.com/23052247/60609051-b48a0c00-9db8-11e9-93fd-0a6500af4e01.png)

It would be nice to have documentation updated or a bug fixed.

**Comment by MightyPen** [comment]
@pensivebrian Hi Brian, can you please look at the Issue #2430 that @IvanAlekseev has encountered with 'SqlPackage.exe'.
@yualan You are listed inside the article as another person who might have knowledge in this area and be able to help.

@stevestein Hi Steve, I am co-assigning this Issue 2430 to you, as the only Writer assigned. If necessary, please assist or prompt Brian and Alan. The eventual outcome might call for a small additional section to add to the article.

**Comment by pensivebrian** [comment]
This looks like a sqlpackage bug. We're looking into it. @IvanAlekseev - it would be helpful to know which connection string name/values are specified in the profile.xml. Are you setting the user id and password in the profile.xml in addition to command line?

**Comment by kisantia** [comment]
@IvanAlekseev we have fixed the bug. The fix will be in the DacFx 18.3 release in a month or two.

**Comment by stevestein** [comment]
@IvanAlekseev - thanks for reporting this. We're going to go ahead and close this issue. Feel free to reach out if needed.

Thanks!
